src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Draw {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public double calculateArea() {
+        return 0;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("message");
```

Comment:
The `draw` method currently prints a generic message. Consider making this method abstract or providing a more meaningful default implementation.